starter-code/app.js

```diff
@@ -1,5 +1,30 @@
+"use strict";
 const express = require('express');
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require('chucknorris-io');
 const client = new Chuck();
 
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+app.get('/random', (req, res, next) => {
+
+  client.getRandomJoke().then((joke) => {
+
+    res.render('index', joke);
+  }).catch((err) => {
+    console.log('ur fucked ' + err);
+  });
+});
+
+app.get('/categories', (req, res, next) => {
+  client.getJokeCategories().then((response) => {
+    const data = {
```

Comment:
Here you are doing what you needed to do in the first get, wrap the information you want to pass to the render in an objetc. Good job.